web/src/lib/components/memory-page/memory-viewer.svelte

```diff
@@ -112,7 +112,7 @@
         </p>
       </svelte:fragment>
 
-      {#if !galleryInView}
+      {#if currentMemory.assets.length > 1} <!-- Display play bar only if there's more than 1 image -->
```

Comment:
We should still show this when there are more memories next. The comment adds no value in my opinion and is better left out.

```suggestion
      {#if canGoForward}
```